README.md

```diff
@@ -102,6 +104,43 @@ An opinionated, highly specific, Elixir wrapper around brod: the Erlang Kafka cl
       worker(Kaffe.Consumer, [])
       ```
 
+### Kaffe GroupMember - Batch Message Consumer
+
+  1. There are two modes for using the batch consumer. The first enables
+     an easy migration path from using the `Kaffe.Consumer`. The second
+     is more batch oriented, which allows consumers to handle a group of
+     messages for efficiency or greater context.
+  
+     `handle_message/1` This is the same as for `Kaffe.Consumer`
+
+     `handle_messages/1` This function (note the pluralization) will be
+     called with a *list of messages*, with each message as a map. Each
+     message map will include the topic and partition in addition to the
+     normal Kafka message metadata.
+
+     The module's `handle_messages/1` function _must_ return `:ok` or
+     Kaffe will throw an error. In normal (synchronous consumer)
+     operation the Kaffe consumer will block until your
+     `handle_messages/1` function returns `:ok`.
```

Comment:
Is there an asynchronous version of `handle_messages`? If so then we need to expand the async consumer section with its details.